### ISSUE TITLE
FS tests are failing on Windows

### ISSUE THREAD

**Issue by kittaakos** [theia-ide/theia#368]
```
  48 passing (4s)
  1 failing

  1) NodeFileSystem #13 watchFileChanges Should receive file changes events from in the workspace by default.:

      Uncaught AssertionError [ERR_ASSERTION]: [ { uri: URI { codeUri: [Object] }, type: 1 },
  { uri: URI { codeUri: [Object] }, type: 1 },
  { uri: URI { codeUri: [Object] } deepEqual [ { uri: URI { codeUri: [Object] }, type: 1 },
  { uri: URI { codeUri: [Object] }, type: 1 },
  { uri: URI { codeUri: [Object] }
      + expected - actual

               "_authority": ""
               "_formatted": [null]
               "_fragment": ""
               "_fsPath": [null]
      -        "_path": "/C:/Users/KITTAA~1/AppData/Local/Temp/node-fs-root/7eef8883-7791-11e7-94d5-015fd19b85c9/foo"
      +        "_path": "/c:/Users/KITTAA~1/AppData/Local/Temp/node-fs-root/7eef8883-7791-11e7-94d5-015fd19b85c9/foo"
               "_query": ""
               "_scheme": "file"
             }
           }
               "_authority": ""
               "_formatted": [null]
               "_fragment": ""
               "_fsPath": [null]
      -        "_path": "/C:/Users/KITTAA~1/AppData/Local/Temp/node-fs-root/7eef8883-7791-11e7-94d5-015fd19b85c9/foo/bar"
      +        "_path": "/c:/Users/KITTAA~1/AppData/Local/Temp/node-fs-root/7eef8883-7791-11e7-94d5-015fd19b85c9/foo/bar"
               "_query": ""
               "_scheme": "file"
             }
           }
               "_authority": ""
               "_formatted": [null]
               "_fragment": ""
               "_fsPath": [null]
      -        "_path": "/C:/Users/KITTAA~1/AppData/Local/Temp/node-fs-root/7eef8883-7791-11e7-94d5-015fd19b85c9/foo/bar/baz.txt"
      +        "_path": "/c:/Users/KITTAA~1/AppData/Local/Temp/node-fs-root/7eef8883-7791-11e7-94d5-015fd19b85c9/foo/bar/baz.txt"
               "_query": ""
               "_scheme": "file"
             }
           }

      at Timeout._onTimeout (src\node\node-filesystem.spec.ts:769:24)



error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**Comment by kittaakos** [comment]
Move test randomly fails on Windows:

```
1) NodeFileSystem 04 #move Moving a non-empty directory to an empty directory. Source folder and its content should be moved to the target location.:
lerna ERR! execute      Error: Timeout of 2000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves.
lerna ERR! execute
```

**Comment by kittaakos** [comment]
Closed via #381.
